frontend/components/commons/forms/questions-form/QuestionsForm.component.vue

```diff
@@ -251,121 +242,103 @@ export default {
         default:
       }
     },
-    onChange({ newOptions, idComponent }) {
+    onChangeTextArea({ newOptions, idComponent }) {
+      this.inputs = this.inputs.map((input) => {
+        if (input.id === idComponent) {
+          input.options = [{ ...newOptions, value: newOptions.text }];
+        }
+        return input;
+      });
+    },
```

Comment:
I think we don't need a map for this:

```suggestion
      const component = this.inputs.find(input => input.id === idComponent);
      component.options = [{ ...newOptions, value: newOptions.text }];
    },
```

Reply:
I pass also the text to update with the standard options shape